src/inc-lib-init.php

```diff
@@ -1345,16 +1346,17 @@ function lovd_getVariantInfo ($sVariant, $sTranscriptID = '', $bCheckHGVS = fals
             $sIntronicLast  = str_replace('_', '_intronic_', $sLast);
 
             if ($aVariant[$sFirst] > $aVariant[$sLast]) {
-                list($aVariant[$sFirst], $aVariant[$sLast]) = array($aVariant[$sLast], $aVariant[$sFirst]);
+                list($aVariant[$sFirst], $aVariant[$sIntronicFirst], $aVariant[$sLast], $aVariant[$sIntronicLast]) =
+                    array($aVariant[$sLast], $aVariant[$sIntronicLast], $aVariant[$sFirst], $aVariant[$sIntronicFirst]);
```

Comment:
This fixes the bug which failed to switch the intronic values along with the 'basic' values.